packages/react-components/src/shared/src/focusableTreeWalker.ts

```diff
@@ -14,21 +14,25 @@ const FocusableElement = [
     "audio[controls]",
     "video[controls]",
     "[contenteditable]"
-];
+] as const;
```

Comment:
I am not sure why you have to do this. This code have been copied from Spectrum which is in TS and they don't have to use `as const`.

See https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/focus/src/FocusScope.tsx#L152

Reply:
Your right, not needed